### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by svarocode** [vercel/community#6692] [Account Issue] [answered]
### Old account, phone in use, different phone result in 500

There is Vercel account associated with my email, so i can't login with github.
There is already account associated with my phone number (my personal account) so i can't login with email.
I need to login with company email I used to login with over a year ago and make a change to a live project, that I can't currently access. 
I am also unable to directly contact Vercel support, because I need to login to do that.
Please help!

After trying with the phone number of colleague - 
{
  "error": {
    "code": "internal_server_error",
    "message": "An unexpected internal error occurred"
  }
}

**Comment by amyegan** [comment, marked as answer]
Hey @svarocode. We can't safely discuss account information in public due to privacy and security concerns so I can't get enough info to fully solve login issues from here. Please email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the team that can help with login issues.

Be sure to let them know the email addresses used for your accounts and how you use each account (i.e. one is for work and the other is for personal projects). That will help them locate and verify both accounts. 🙏

https://vercel.com/guides/how-to-get-vercel-support#direct-channels

**Reply by amyegan** [reply]
We also released a change to let people associate their personal and work emails with a single Vercel account so it's no longer necessary to have two accounts.

https://vercel.com/changelog/accounts-can-now-have-multiple-email-addresses
